jenkins-slaves/jenkins-slave-mvn/Dockerfile

```diff
@@ -1,2 +1,2 @@
-FROM quay.io/openshift/origin-jenkins-agent-maven:4.2
+FROM quay.io/openshift/origin-jenkins-agent-base:4.4
```

Comment:
hmm, how did this pass CI? In the past there was no `mvn` in the `origin-jenkins-agent-base`. Maybe it changed but I'd like to make sure first (and also figure out if there's an issue with our CI).

Reply:
I put it back to the mvn slave anywhos. From memory the base all have Java installed so no surprise to find maven hanging around in there.